lib/stronger_parameters/constraints/file_constraint.rb

```diff
@@ -0,0 +1,13 @@
+require 'stronger_parameters/constraints'
+
+module StrongerParameters
+  class FileConstraint < Constraint
+    def value(v)
+      if [File, StringIO].include? v.class
```

Comment:
`v.is_a?(File) || v.is_a?(StringIO)` would be better. Then you also catch subclasses.